tests/v1/generation/test_batch_invariance.py

```diff
@@ -516,8 +519,23 @@ def test_logprobs_WITHOUT_batch_invariance_should_FAIL(backend):
             dtype="bfloat16",
         )
 
-        # Use more realistic prompts for better token generation
-        prompts = [_random_prompt(10, 50) for i in range(32)]
+        # build ragged prompts to change shapes significantly across BS=1 vs BS=N
+        long_min = int(os.getenv("VLLM_MIN_PROMPT", "768"))
+        long_max = int(os.getenv("VLLM_MAX_PROMPT", "2048"))
+        prompts: list[str] = []
+        for i in range(32):
+            if i % 4 == 0:
```

Comment:
maybe `random.randint(0,4) % 4` instead of 4?  this kind of guarantees a structure